src/cdi_CPEloss/Analytic_Constant_Eloss_Model.hh

```diff
@@ -0,0 +1,52 @@
+//----------------------------------*-C++-*-----------------------------------//
+/*!
+ * \file   cdi_CPEloss/Analytic_Constant_Eloss_Model.hh
+ * \author Ben R. Ryan
+ * \date   Feb 21 2020
+ * \brief  Analytic_Constant_Eloss_Model class definition.
+ * \note   Copyright (C) 2016-2020 Triad National Security, LLC.
+ *         All rights reserved. */
+//----------------------------------------------------------------------------//
+
+#ifndef rtt_cdi_CPEloss_Analytic_Constant_Eloss_Model_hh
+#define rtt_cdi_CPEloss_Analytic_Constant_Eloss_Model_hh
+
+#include "Analytic_Eloss_Model.hh"
+#include <algorithm>
+
+namespace rtt_cdi_cpeloss {
+//============================================================================//
+/*!
+ * \class Analytic_Constant_Eloss_Model
+ * \brief Analytic_Constant_Eloss_Model derived class.
+ *
+ * CP energy loss class using constant value provided to constructor.
+ */
+//============================================================================//
+
+class Analytic_Constant_Eloss_Model : public Analytic_Eloss_Model {
+public:
+  //! Constructor
+  Analytic_Constant_Eloss_Model(const rtt_cdi::CParticle &target,
+                                const rtt_cdi::CParticle &projectile,
+                                const double eloss_value_in)
+      : Analytic_Eloss_Model(target, projectile), eloss_value(eloss_value_in) {}
+
+  //! Calculate the eloss rate in units of shk^-1;
+  //! T given in keV, rho in g/cc, v0 in cm/shk
+  double calculate_eloss(const double T, const double rho,
```

Comment:
I think we typically comment out the unused variable names in cases like this, (eg. "const double T" -> "const double /*T*/") to avoid warnings... although I also don't see any warnings... maybe because nothing in draco includes this file yet?

Reply:
Good catch! Indeed, when I add this file to a test,
```
In file included from /home/brryan/github/Draco/src/cdi_CPEloss/test/tstAnalytic_CP_Eloss.cc:13:
/home/brryan/github/Draco/src/cdi_CPEloss/Analytic_Constant_Eloss_Model.hh: In member function 'virtual double rtt_cdi_cpeloss::Analytic_Constant_Eloss_Model::calculate_eloss(double, double, double) const':
/home/brryan/github/Draco/src/cdi_CPEloss/Analytic_Constant_Eloss_Model.hh:37:39: warning: unused parameter 'T' [-Wunused-parameter]
   37 |   double calculate_eloss(const double T, const double rho,
      |                          ~~~~~~~~~~~~~^
/home/brryan/github/Draco/src/cdi_CPEloss/Analytic_Constant_Eloss_Model.hh:37:55: warning: unused parameter 'rho' [-Wunused-parameter]
   37 |   double calculate_eloss(const double T, const double rho,
      |                                          ~~~~~~~~~~~~~^~~
/home/brryan/github/Draco/src/cdi_CPEloss/Analytic_Constant_Eloss_Model.hh:38:39: warning: unused parameter 'v0' [-Wunused-parameter]
   38 |                          const double v0) const override {
      |                          ~~~~~~~~~~~~~^~
```

Reply:
As @keadyk indicates, this file isn't actually compiled, so our CI is only doing the clang-format style check.  Please add a unit test or extend an existing test.

Reply:
Done!